agent/containers/images/Makefile

```diff
@@ -108,7 +108,7 @@ _ALL_RPMS = ${_TOOL_RPMS} ${_WORKLOAD_RPMS}
 # By default we only build images for the following distributions.
 _DEFAULT_DISTROS = \
 	centos-9 centos-8 centos-7 \
-	fedora-36 fedora-35
+	fedora-38 fedora-37 fedora-36
 #	rhel-9 rhel-8 rhel-7
```

Comment:
Do want to still continue to retain this commented line?

Reply:
Um, yes?  😇

We have the machinery for building for RHEL.  We build RHEL-based containers for building the RHEL RPMs, which we push to the internal registry, and we use them for building RHEL RPMs.  But, we don't build RHEL-based versions of the Agent containers out of uncertainty as to whether we can redistribute them.  So, that line is "aspirational"...as an irritant to resolve the question of whether we can/should build them.